Formula/tree-sitter.rb

```diff
@@ -15,10 +17,19 @@ class TreeSitter < Formula
     sha256 "fbef61fe89cc07c5af05a06185ccb91cfa9c34773ab6b26b751ae0171e3806a2" => :mojave
   end
 
+  depends_on "emscripten" => [:build, :test]
+  depends_on "node" => [:build, :test]
   depends_on "rust" => :build
-  depends_on "node" => :test
 
   def install
+    cd "lib/binding_web" do
+      system "npm", "install", *Language::Node.local_npm_install_args
+    end
+    system "script/build-wasm"
+    %w[tree-sitter.js tree-sitter-web.d.ts tree-sitter.wasm package.json].each do |file|
+      (lib/"binding_web").install "lib/binding_web/#{file}"
+    end
```

Comment:
Thanks, @paperchalice. One last thing: can you move this to the end of the install block?